presets/testnet/network.yml

```diff
@@ -32,6 +32,8 @@ votingKeyLinkV2: 215500
 importanceBlock: 215500
 accountRestrictionsV2: 215500
```

Comment:
Are we still using v2?
Should all be reset to v1 now I think

Reply:
We will eventually